.github/workflows/test_with_tox.yml

```
@@ -66,7 +66,7 @@ jobs:
         runs-on: ${{ matrix.os }}
 
         strategy:
-            fail-fast: true
+            fail-fast: false
```

```suggestion
            fail-fast: true
```

apply before merging